src/Greeting.js

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchRandomGreeting } from '../store/greetingReducer.js';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const randomGreeting = useSelector((state) => state.greeting);
+
+  useEffect(() => {
+    dispatch(fetchRandomGreeting());
+  }, [dispatch]);
+
+  return (
+    <div>
+      <h1>Random Greeting:</h1>
+      <p>{randomGreeting}</p>
+    </div>
+  );
+};
+
+export default Greeting;
```

Comment:
- [ ] KIndly Delete this component as it is not used, this will clean you project